plugins/pencilblue/templates/angular/admin/content/media/manage_media.html

```diff
@@ -33,8 +33,25 @@
 			sortDesc: false
 		}];
 
+		$scope.setItem = function(item) {
+				$scope.topic = item;
+				$scope.search();
+				$scope.$apply();
```

Comment:
it's was copy/paste from [here](https://github.com/pencilblue/pencilblue/blob/master/plugins/pencilblue/templates/angular/admin/content/navigation/nav_item_form.html#L13). I decided will be problem with popmenu if i remove this code.